src/lib/Layout/PanelSideBarLayout/PanelSideBar/PanelSideBarItem.tsx

```diff
@@ -53,6 +53,7 @@ const PanelSideBarItem = <TPanelItemId extends string, TPanelItem>(props: PanelS
     if (scrollToActiveItemRef.current && isActive) {
       scrollToActiveItemRef.current.scrollIntoView();
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
add it to deps instead of eslint-disable

Reply:
done